build/automation/lib/k8s.mk

```diff
@@ -227,6 +227,19 @@ k8s-job-has-failed: ### Show whether the job failed - return: [true|""]
 		--output jsonpath='{.status.conditions[?(@.type=="Failed")].status}' \
 	| tr '[:upper:]' '[:lower:]' | tr -d '\n'
 
+k8s-run-cronjob-as-job: ### run a cronjob as a job (now) - mandatory: K8S_CRONJOB_NAME=[name], PROFILE=[name]; optional K8S_JOB_NAME=[job name];
```

Comment:
Can we change target signature to `k8s-job-run-cronjob-now: ### Run a cronjob now - mandatory: NAME=[name],PROFILE=[name]`, please?

Reply:
fixed